Sources/SwiftGRPC/Core/Channel.swift

```diff
@@ -132,18 +132,17 @@ public class Channel {
 }
 
 private extension Channel {
-  class ConnectivityObserver {
+  final class ConnectivityObserver {
     private let completionQueue: CompletionQueue
     private let underlyingChannel: UnsafeMutableRawPointer
-    private let underlyingCompletionQueue: UnsafeMutableRawPointer
+    private let underlyingCompletionQueue: UnsafeMutableRawPointer = cgrpc_completion_queue_create_for_next()
```

Comment:
Not sure I like this move, given that it is closely related to the initialization of `self.completionQueue`.

Reply:
Fair, reverted